app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(comand: str) -> None:
+    names = comand.split(" ")
+    if names[1] == names[2]:
+        print("Does nothing")
```

Comment:
it's redundant condition and print 

![image](https://user-images.githubusercontent.com/104218550/220344043-5e305a04-e737-4361-9643-3d87e0e8acb6.png)